scripts/get-fsc-tests.sh

```diff
@@ -0,0 +1,15 @@
+#!/bin/bash
+# This script fetches Full stack compatibility suite and copies all the feature files into root/tests/integration/features directory
+# and copies all the datafiles into fsc-datafiles folder.
+set -e
+FSC_PATH=tmp/fsc-repo
+rm -rf $FSC_PATH
+mkdir -p $FSC_PATH
+ 
+pushd $FSC_PATH && git init && git fetch --depth=1 https://$CI_USER_TOKEN@github.com/optimizely/fullstack-sdk-compatibility-suite ${FSC_BRANCH:-master} && git checkout FETCH_HEAD && popd
+mkdir -p ./tests/integration/features
+cp -r ./$FSC_PATH/features/* ./tests/integration/features
+mkdir -p ./fsc-datafiles
+cp -r ./$FSC_PATH/features/support/datafiles/*.json ./fsc-datafiles
+export DATAFILES_DIR=$TRAVIS_BUILD_DIR/fsc-datafiles
```

Comment:
This is specific to just Travis, can it be done in a way that is agnostic or do we need to know about the `TRAVIS_BUILD_DIR` variable? Maybe we can set a generic var in the travis file that this references